### PR TITLE
wip -- add script to publish versioned snapshot of docs to `rook.github.io`

### PR DESCRIPTION
**Pull request by ilovett**

### CODE REVIEW

**Comment by jbw976** [comment]
@ilovett the big #700 PR went into master and there's a number of conflicts you'll need to fix here now.

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,7 @@
 /.work
 /.download
 /.idea
+/.tmp
```
Comment (bassam) [inline .gitignore]: can you use `.work` instead? it already ties into the Makefiles

File: build/docs.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -e
```
Comment (bassam) [inline build/docs.sh]: can this go in build/release/docs.sh and follow the convention there?

Comment (bassam) [inline build/docs.sh]: more comments here https://github.com/rook/rook/issues/549#issuecomment-307650999

File: .gitignore (hunk 1 above)
Comment (ilovett) [inline .gitignore]: addressed

File: build/docs.sh (hunk 2 above)
Comment (ilovett) [inline build/docs.sh]: moved `docs.sh` to `release` dir but still need to implement to match convention of the other scripts

**Comment by bassam** [comment]
closing in favor of #783

**Pull request closed without merging**